public/index.jade

```diff
@@ -1,48 +1,7 @@
-!= partial("components/_hero", public._data.current )
-
-if public._data.current.offmonth
-
-    div.jsla-toplevel-container
-
-        div.off-month.lightning
-
-            h2.off-month-title We are off this month
-            p.description= public._data.current.description
-
-            blockquote Get notified when the next event is by joining the
-                a(href="http://eepurl.com/jdRNb") Newsletter
-
-
-        if public._data.current.videos
-
-            div.featured-videos.lightning
-                h2.off-month-title If you missed us in October, the videos are now available.
-
-                each video in public._data.current.videos
-
-                    div.featured-video
-                        h5.video-title= video.title
-                        a.fancybox-media(href="http://vimeo.com/#{video.video}")
-                            img.videoimg(src="/images/videos/#{video.videoimg}", width="500", height="281")
-
-
-        if public._data.current.events
-
-            div.featured-events.lightning
-                h2.off-month-title Other awesome LA events.
-                ul.events
-                    // pretty much a list of links
-                    each _event in public._data.current.events
-                        li.event
-                            a( href=_event.link )
-                                h6= _event.title
-
-
-        != partial("_want-to-present")
-
-
-else
 
+if public._data.current.datetime
```

Comment:
I guess just setting the datetime to `null` would be just as effective as the flag I was talking about above.

Reply:
Just now thinking we could also do some date math for this. If the current datetime has passed (relative to the date now, as in at build time) then show the no event page.

This would mean all it would take is a rebuild to get the TBA page up.

Reply:
Yeah I have a few thoughts around this. It would be awesome to have some of these switch be more automatic. The building of static pages kinda makes it difficult.